pyomo/contrib/pynumero/cmake/asl_interface/src/AmplInterface.cpp

```diff
@@ -12,6 +12,7 @@
 #include "AmplInterface.hpp"
 #include "AssertUtils.hpp"
 #include "asl_pfgh.h"
+//#include "als.h"
```

Comment:
This seems like an odd header to comment out.  Is it implied by `asl_pfgh.h`?  If it is not needed, shouldn't it just be deleted?

Reply:
Yeah, I left that behind on accident. Removing...